src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
Well, `ActionList` is a bit more than just a plain list of rows, so I would introduce `<DraggableList />` instead of `<div {...styling('actionListRows')} ref='rows'>` that would be responsible just for dragging, so it would be like:
```jsx
<DraggableList onReorderAction={...} ...>
 {ids.map(id => <DraggableListRow ... ><ActionListRow ... /></DraggableListRow>}
</DraggableList>
```